WindowsServerDocs/identity/ad-cs/certificates-and-trust.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Trusted roots and disallowed certificates
+description: Learn how the Microsoft Root Certificate Program works to distribute trusted root certificates automatically across Windows operating systems in disconnected environments.
+ms.service: 
+author: gswashington
+ms.author: roharwoo
+ms.topic: overview
+ms.date: 01/18/2023
+
+---
+
+# Certificates and trust
+
+Applies To: Windows Server (All supported versions), Windows clients, Azure Stack HCI.
+
+The Microsoft Root Certificate Program enables distribution of trusted and untrusted root certificates within Windows operating systems. For more information about the list of members in Windows Root Certificate Program, see [Windows Root Certificate Program - Members List (All CAs)](GET VERIFIED LINK FROM FLORIN).
+
+Trusted and untrusted root certificates are used by Windows operating systems and applications as a reference when determining whether public key infrastructure (PKI) hierarchies and digital certificates are trustworthy. Untrusted root certificates are certificates that are publicly known to be fraudulent. Trusted and untrusted root certificates functionality works across all environments, whether connected or disconnected.
+
+Trusted and untrusted root certificates are contained in a certificate trust list (CTL). When you want to distribute root certificates you use a CTL. Windows Server features automatic daily update functionality that includes downloads of latest CTLs. The list of trusted and untrusted root certificates are called the Trusted CTL and Untrusted CTL, respectively. See [Announcing the automated updater of untrustworthy certificates and keys](https://blogs.technet.com/b/pki/archive/2012/06/12/announcing-the-automated-updater-of-untrustworthy-certificates-and-keys.aspx) for more information.
+
+Servers and clients access the Windows Update site to update the CTL using the automatic daily update mechanism (CTL updater) discussed in this article. You can take advantage of CTL updater functionality by installing the appropriate software updates. See [Configure Trusted Roots and Disallowed Certificates](configure-trusted-roots-disallowed-certificates.md) for guidance in installing the software updates on supported operating systems discussed in this article.
+
+## Automatic certificate trust list updates
+
+By default, Windows downloads the CTLs from the Internet via an automatic mechanism called the CTL Updater. The public URLs used by the CTL Updater can be made available to clients. These are:
```

Comment:
` These are:` - These isn't always clear to the reader when English is their second language or the text is localized. Could you use a noun or rephase?